### DISCUSSION TITLE
Is it possible to make a custom close button appear onMouseOver of the ToastContainer?

### DISCUSSION THREAD

**Discussion by Adam-Gemperline** [fkhadra/react-toastify#780] [Q&A]
Is it possible to make a custom close button appear onMouseOver of the ToastContainer? The ToastContainer seems to prevent re-rendering of the close button after state changes have taken place. 

I haven't found any working examples of this. If anyone has an answer to this question or has seen similar solutions, please let me know! Thanks :)

**UPDATE:** 
Custom close button is working on 'hover' but only one instance of the button is available and it is shown on the most recently created toast. I have set my ToastContainer limit to 3, so when there is more than one active notifications visible, the close button only appears on the bottom (most-recent) notification when hovering on any of them.

**Comment by tzztson** [comment]
Hello, Adam.

I am not sure it is possible to make custom close button using ToastContainer.

But I've experience to make custome close button using core CSS before.

Please contact me if you need my help.

Thank you.
